### PR TITLE
Login debug

### PR DESCRIPTION
**Pull request by aaronfinsrud**
## Checklist:
[x] Improved error handling in Login component
[x] Removed unnecessary functions from context wrapper component
[x] Removed useMemo from context wrapper component

### CODE REVIEW

File: client/src/components/Login.jsx [hunk 1]
```diff
@@ -32,11 +32,9 @@ function Login() {
     e.preventDefault();
     try {
       setLoading(true);
-      await login(email, password)
-        .then((userCredential) => {
-          const { user } = userCredential;
-          console.log('user cred', userCredential);
-          // TODO: add userId to context
+      login(email, password)
```
Comment (alexander-huerta) [inline client/src/components/Login.jsx:35]: how will we be storing the user credentials?

File: client/src/contexts/index.jsx [hunk 2]
```diff
@@ -44,18 +41,6 @@ export function AuthProvider({ children }) {
     return signOut(auth);
   }
 
-  function resetPassword(email) {
```
Comment (alexander-huerta) [inline client/src/contexts/index.jsx:47]: Did you decide against incorporating reset password, update email/password for this first iteration of the app?

**Review by alexander-huerta — APPROVED** [review]
Approving this. I left a few questions though. We can discuss later

File: client/src/components/Login.jsx (hunk 1 above)
Comment (aaronfinsrud) [reply client/src/components/Login.jsx:35]: They're stored on the firebase authentication db

File: client/src/contexts/index.jsx (hunk 2 above)
Comment (aaronfinsrud) [reply client/src/contexts/index.jsx:47]: Yeah - I figure it will keep things cleaner in the context/index.js file

**Pull request merged**